pkg/api/common/versions.go

```diff
@@ -269,6 +269,7 @@ var AllKubernetesSupportedVersionsAzureStack = map[string]bool{
 	"1.17.7":  false,
 	"1.17.9":  true,
 	"1.17.11": true,
```

Comment:
FYI, the linux VHD is not pre-pulling v1.17.11, not sure we want to keep both v1.17.11 and v1.17.17

Reply:
Good catch, I did not notice it, will submit a commit to remove v1.17.11